components/atoms/BenefitCardHeaderActive.js

```diff
@@ -25,15 +25,11 @@ export default function BenefitCardHeaderActive(props) {
           {props.benefit.applicationStatus}
         </h2>
       </div>
-
-      <div className="mx-auto sm:grid sm:grid-cols-4 sm:divide-x-2">
-        <div id="paymentStartDate" className="col-span-1 py-4 sm:px-3">
-          <div className="font-bold font-display text-5xl mb-2">
-            {props.benefit.benefitType}
-          </div>
-          <p className="font-display font-bold text-lg">
+      <div className="sm:flex sm:justify-around sm:grid sm:grid-cols-4 divide-x-2 sm:mx-auto">
```

Comment:
Is there a reason grid and flex are applied to the same `<div>`? I think the spacing can be accomplished with just flex.